bitcoin-wallet/src/wallet.rs

```diff
@@ -2884,1190 +2910,97 @@ impl TestWalletBuilder {
     }
 }
 
-#[cfg(test)]
-mod tests {
```

Comment:
can you confirm that all tests were moved somewhere (even if they'd fail) such that we can get them working at a later time without losing them?

Reply:
Yeah, they all get moved and fixed up, I didn't delete any